h2o-core/src/main/java/water/fvec/NewChunk.java

```diff
@@ -1537,7 +1542,7 @@ private Chunk chunkUUID() {
       if(idx >= 0)i = idx;
       else cancel_sparse(); // for now don't bother setting the sparse value
     }
-    _is[i] = _sslen;
+    if (_is != null) _is[i] = _sslen;
```

Comment:
This doesn't seem correct to me. If _ss gets modified (in append_ss function) then _is needs to be modified as well because "_is[] holds offsets into _ss[]".

Can you please explain this change?